tinker/faculty_bios/__init__.py

```diff
@@ -215,3 +217,100 @@ def edit_all(self):
         xml_url = app.config['FACULTY_BIOS_XML_URL']
         self.base.edit_all(type_to_find, xml_url)
         return 'success'
+
+    @route("/faculty-bio-csv")
+    def get_faculty_bio_csv(self):
```

Comment:
You also are maintaining 2 lists of the keys. It might be nice to set up the max list in one place - if possible.

Reply:
Done I believe.